docs/detections/api/rules/rules-api-delete.asciidoc

```diff
@@ -2,6 +2,7 @@
 === Delete rule
 
 Deletes a single rule using the `rule_id` or `id` field.
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Comment:
This looks a bit off. May all we need is a new line to fix it?
![image](https://user-images.githubusercontent.com/616158/118678682-13d94580-b7b2-11eb-9a72-f95e84874070.png)